### PR TITLE
Add Search Input

### PR DESCRIPTION
**Pull request by SaraVieira**
* Adds search input
* Adds more onChange test cases

### CODE REVIEW

**Comment by lbogdan** [comment]
Build for latest commit https://github.com/codesandbox/codesandbox-client/pull/3297/commits/56827a04ff921cb7978a76fb477719b917013d0e is at https://pr3297.build.csb.dev/s/new.

File: packages/common/src/design-language/theme.ts [hunk 1]
```diff
@@ -7,6 +7,16 @@ const theme = {
   fontWeights,
 
   // we use a 4 point grid
+  // 0 - 0
+  // 1 - 4
+  // 2 - 8
+  // 3 - 12
+  // 4 - 16
+  // 5 - 24
```
Comment (siddharthkp) [inline packages/common/src/design-language/theme.ts]: this isn't correct. 5 * 4 = 20

**Comment by siddharthkp** [comment]
Do you think we should use <Input type="search"/> instead of a new component?

File: packages/common/src/design-language/theme.ts (hunk 1 above)
Comment (SaraVieira) [reply packages/common/src/design-language/theme.ts]: Not on my watch!

Comment (siddharthkp) [reply packages/common/src/design-language/theme.ts]: but but?

**Pull request merged**